tests/app/celery/test_tasks.py

```diff
@@ -440,12 +440,12 @@ def test_should_process_all_sms_job(sample_job_with_placeholdered_template, mock
         (LETTER_TYPE, True, "save_letter", "research-mode-tasks"),
     ],
 )
-def test_process_row_sends_letter_task(notify_api, template_type, research_mode, expected_function, expected_queue, mocker):
+def test_process_row_sends_save_task(notify_api, template_type, research_mode, expected_function, expected_queue, mocker):
```

Comment:
I renamed this test since it's not just about letters, it's testing that `save_email`, `save_sms` and `save_letter` are called correctly.